dev-python/PyVISA-py/PyVISA-py-0.7.1.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( pypy3 python3_{10..12} )
+PYTHON_COMPAT=( pypy3 python3_{10..13} )
```

Comment:
We're also one version behind for this one. Could you bump it to the latest version while we are touching this?

Reply:
Done, test passes. 
I also noticed that the HOMEPAGE for PyVISA-sim was wrong, so I also fixed that.